cfy_manager/main.py

```diff
@@ -837,12 +844,14 @@ def start(verbose=False):
     _print_time()
 
 
-def stop(verbose=False, force=False):
+@argh.arg('include_components', nargs='*')
+def stop(include_components, verbose=False, force=False):
     """ Stop Cloudify Manager services """
 
-    _prepare_execution(verbose)
+    _prepare_execution(verbose, components=include_components)
     _validate_components_prepared('stop')
-    _validate_force(force, 'stop')
+    if not include_components:
```

Comment:
Well I never fully liked this protection against mistakes, but I think it still applies here

Reply:
i don't like it either, and therefore i see no reason to have it :P
`systemctl restart cloudify-restservice` doesn't need a `--force` either, why should we ;)

Reply:
If you mean having to append --force, yeah- I don't think it's that good protection anyway, since it requires you to put a command line argument in, so you just end up doing that as part of running the command. It doesn't protect from running something from bash history either, so... I'm not sure what it's for, really.

Reply:
I agree :)